tests/py/test_fake_data.py

```diff
@@ -34,3 +33,12 @@ def test_fake_participant_identity(self):
         crusher = self.make_participant('crusher', email_address='crusher@example.com')
         country_id = fake_data.fake_participant_identity(crusher)
         assert [x.country.id for x in crusher.list_identity_metadata()] == [country_id]
+
+    def test_fake_team(self):
+        """
+            Just a test to ensure that fake_team does not fail for 
+            teams with apostrophes
+        """
```

Comment:
It's not obvious from this test file, but generally instead of a docstring we make the test name more verbose, a sentence describing the test.

```python
def test_that_thing_does_what_its_supposed_to(self):
```